.gitignore

```diff
@@ -1,3 +1,4 @@
+.env
```

Comment:
this line can be removed - there is no top-level .env file

Reply:
Yeah, I already removed it